app/javascript/components/rules/forms/RuleForm.vue

```diff
@@ -399,6 +413,7 @@ export default {
   data: function () {
     return {
       mod: Math.floor(Math.random() * 1000),
+      ruleCopy: _.cloneDeep(this.rule),
```

Comment:
I'm not convinced on ruleCopy being an actual data component. It seems like it would be better as a computed property with a getter and a setter like: https://github.com/mitre/heimdall2/blob/b2daeb47ba457b67a8881bd8914a4c76a7e9c5b7/apps/frontend/src/components/global/SearchBar.vue#L57-L63

https://vuejs.org/v2/guide/computed.html#Computed-Setter

Reply:
This has been changed so that neither a data or computed property are required in the forms.